index.html

```diff
@@ -16,42 +16,246 @@
+              <div class="frame">
+                <h5>CANOPY</h5>
+                <ul>
+                  <li>Back End Dev</li>
+                  <li>2015</li>
+                </ul>
+              </div>
+              <p class="paragraph">
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </p>
+              <div class="languages">
+                <ul class="language">
+                  <li>html</li>
+                  <li>css</li>
+                  <li>javascript</li>
+                </ul>
+              </div>
+              <div class="btn"><button>See Project</button></div>
+            </div>
+          </div>
+        </div>
+        <div>
+          <div class="grid-item">
+            <div class="card">
+              <div class="project">
+                <img src="/images/second-project.png" alt="my second project" />
+              </div>
+
+              <h2 class="project-title">Tonic</h2>
+              <div class="frame">
+                <h5>CANOPY</h5>
+                <ul>
+                  <li>Back End Dev</li>
+                  <li>2015</li>
+                </ul>
+              </div>
+              <p class="paragraph">
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </p>
+              <div class="languages">
+                <ul class="language">
+                  <li>html</li>
+                  <li>css</li>
+                  <li>javascript</li>
+                </ul>
+              </div>
+              <div class="btn"><button>See Project</button></div>
+            </div>
+          </div>
+        </div>
+        <div>
+          <div class="grid-item">
+            <div class="card">
+              <div class="project">
+                <img src="images/third-project.png" alt="my third project" />
+              </div>
+
+              <h2 class="project-title">Tonic</h2>
+              <div class="frame">
+                <h5>CANOPY</h5>
+                <ul>
+                  <li>Back End Dev</li>
+                  <li>2015</li>
+                </ul>
+              </div>
+              <p class="paragraph">
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </p>
+              <div class="languages">
+                <ul class="language">
+                  <li>html</li>
+                  <li>css</li>
+                  <li>javascript</li>
+                </ul>
+              </div>
+              <div class="btn"><button>See Project</button></div>
+            </div>
+          </div>
+        </div>
+        <div>
+          <div class="grid-item">
+            <div class="card">
+              <div class="project">
+                <img src="/images/fouth-project.png" alt="my first project" />
+              </div>
```

Comment:
Good job so far 👍. Kindly check your image path, it seems it's broken and the image is not displaying. Also, kindly remove unnecessary spaces and let your code be in line with the Figma design 👇
<img width="382" alt="image" src="https://user-images.githubusercontent.com/73854999/205440689-ef5af6d4-c858-4df9-8dc9-1c79e82f3b15.png">

Reply:
@Lizdev-05, the error in the path has been corrected.